Implementing Screening Feedback

AlexanderGregorSanchez opened the pull request.

AlexanderGregorSanchez committed:
Converted Ynah Health Regen interval from 7 to 3.5 / changed how Ynah's shadow works

AlexanderGregorSanchez committed:
Updated Item pick up Delay and collider size / Revomed alot of unused sounds

daleabellaa committed:
added attack vfx for teriyaki tempest

AlexanderGregorSanchez committed:
added faceX to fire meat ball / converted meteor to enemies in range + Damage Ticker for player

AlexanderGregorSanchez committed:
added a OnDestroy() for fire meat ball player ability

AlexanderGregorSanchez committed:
Added isBoss check in GroundWave object

AlexanderGregorSanchez committed:
Merge remote-tracking branch 'origin/feature/tempest-vfx' into Implementing_Screening_Feedback

AlexanderGregorSanchez committed:
Added Tempest VFX

AlexanderGregorSanchez committed:
Changed level mask to shrink and grow / minor animation controller stuffs

AlexanderGregorSanchez committed:
Circle and triangle now swap and change colors / LevelMask now grows and shrinks correctly

AlexanderGregorSanchez committed:
Artifact ability availability Icon now see through of not enough stamina / Error SFX on attempt if ability cost higher than stamina added

LauSnchzz committed:
Cooking Popup

- cooking pop up now displays higher

LauSnchzz committed:
Stats Text

- inventory perma and temp text added

LauSnchzz committed:
Crafting Nutrition Text

- added nutrition text on crafting ui

LauSnchzz committed:
added new art assets

- added new updated art assets to address comments

LauSnchzz committed:
Added highlighted words

- added highlighted words on ingredients to emphasize nutrients

LauSnchzz committed:
Added highlights on meal

LauSnchzz committed:
added ynah sounds but not implemented

AlexanderGregorSanchez committed:
Key icons added

AlexanderGregorSanchez committed:
Added goat charge wind up

AlexanderGregorSanchez committed:
Added Anim Hit Checks to all melee attacks

AlexanderGregorSanchez committed:
Added Anim Hit Checks to all melee attacks

AlexanderGregorSanchez committed:
Merge branch 'Implementing_Screening_Feedback' of https://github.com/KoolieAid/CookingYnah into Implementing_Screening_Feedback

LauSnchzz committed:
Ynah attack sounds added

- added ynah attack sounds

LauSnchzz committed:
Added ynah hurt sound

AlexanderGregorSanchez committed:
Added Dungeon music / music now pauses when enter boss room (#53)

Co-authored-by: LauSnchzz <129297636+LauSnchzz@users.noreply.github.com>

LauSnchzz reviewed (approved):
looks good we just need to double check all of the implemented things if there were bugs that reappeared new bugs

AlexanderGregorSanchez committed:
Fixed an issue where artifact script was throwing a null

AlexanderGregorSanchez committed:
Fixed an issue with the music not looping

AlexanderGregorSanchez committed:
turned down the dungeon music so you can hear the environment audio better / Fixed an issue with the hit check was throwing null (missing acrual checking logic)

AlexanderGregorSanchez committed:
fixed an issue with the boss exits sorting layer

daleabellaa committed:
changed dungeon surroundings to black

made the dungeon surroundings black to make the environment pop up more & to give a dark dungeon ambient

AlexanderGregorSanchez committed:
Fixed an issue of OmniSlash damage and animation not working

AlexanderGregorSanchez committed:
Merge branch 'Implementing_Screening_Feedback' of https://github.com/KoolieAid/CookingYnah into Implementing_Screening_Feedback

LauSnchzz committed:
removed artifact 1 check

LauSnchzz committed:
exit to main menu debug

KoolieAid committed:
fixed the twitch debugger for standalone build win64

LauSnchzz committed:
added enemy damage cause i am a dumbass

KoolieAid committed:
forgot i changed this signature lmao
